projects/packages/waf/tests/php/integration/bootstrap.php

```diff
@@ -12,3 +12,9 @@
 
 // Initialize WordPress test environment
 \Automattic\Jetpack\Test_Environment::init();
+
```

Comment:
Something like this maybe?
```suggestion

// Mock version of `wpcom_is_vip()` function for testing `Host::is_vip_site()`.
```

Reply:
Yes good idea, moving too quickly so missed your comment suggestion.